pkg/utils/nad.go

```diff
@@ -54,10 +54,10 @@ func NewLayer3NetworkConf(conf string) (*Layer3NetworkConf, error) {
 	if networkConf.Mode != "" && networkConf.Mode != Auto && networkConf.Mode != Manual {
 		return nil, fmt.Errorf("unknown mode %s", networkConf.Mode)
 	}
-	if _, _, err := net.ParseCIDR(networkConf.CIDR); networkConf.CIDR != "" && err != nil {
+	if _, _, err := net.ParseCIDR(networkConf.CIDR); networkConf.Mode == Manual && networkConf.CIDR != "" && err != nil {
```

Comment:
IIRC, both DHCP and Manual mode would allow specifying the DHCP server.

Reply:
Actually not in UI, only specifying the DHCP server in auto mode.